### ISSUE TITLE
🐛[BUG][Quest/Mission] Sheep in Antlion's Clothing Multiple Issues

### ISSUE THREAD

**Issue by charbsreign** [HorizonFFXI/HorizonXI-Issues#1421]
**What does this bug affect?**
ENM - Sheep in Antlion's Clothing

1. Mobs from a completely different ENM are spawning inside this one.

2. The ENM fully heals HP after its 25% submerge.
**Steps to Reproduce**

Simply enter the ENM and reduce the boss's hp until it submerges at 25%
**Expected behavior**

1. Mobs from other ENM's don't belong here
3. Boss should not fully heal.
**Screenshots**

**Comment by charbsreign** [comment]
![image](https://user-images.githubusercontent.com/29908978/232182149-c166ed6a-4155-48c7-b1ec-2059a32a8169.png)
![Capture](https://user-images.githubusercontent.com/29908978/232182189-6a701ea8-ee08-4bc7-a89b-fa123989dbee.PNG)

**Comment by CriticalXI** [comment]
Do you have retail evidence for the correct behavior for Tuchulcha's HP %?

**Comment by charbsreign** [comment]
> 

I've looked through the wiki and allakhazam and could find no reference of it healing anywhere.

14 year old video:
https://youtu.be/TGbf86CGpHE

Another group on horizon doing the fight and it doesn't heal:
https://youtu.be/OfoY0htNsng

I don't know what's causing it, we theorized having adds up heals it but many strategies call to just CC adds with no mention of it healing. Only other thing I can think of is it's getting a Regen effect while submerged (it's healed on us during 2 different runs for different % of hp) or it's reseting combat and healing.
